lib/src/widgets/view.dart

```diff
@@ -71,6 +79,10 @@ class EditorJSViewState extends State<EditorJSView> {
         styles = EditorJSViewStyles.fromJson(jsonDecode(widget.styles!));
 
         customStyleMap = generateStylemap(styles.cssTags!);
+        customTextStyleMap = generateTextStylemap(styles.cssTags!);
+
+        log("STYLES: " + customStyleMap.toString());

```

Comment:
we should probably remove these logs, I imagine they could get pretty large in the console when we're debugging